### PR TITLE
Fix error messages can't be copied

### LINKED ISSUE #6749: Unable to copy the error message in clipboard
**What you were expecting:**

I expect the error message to be copyable in the clipboard.

image

**What happened instead:**

When I start selecting the error message, it toggles the accordion.

**Related code:**

https://codesandbox.io/s/confident-sanderson-o4igs?file=/src/posts/PostList.tsx

**Environment**

* React-admin version: 3.19.0
* React version: 17.0.0
* Browser: Chrome 95

### PR DESCRIPTION
**Pull request by WiXSL**
Fixes #6749

Before;
140480134-5697c857-b012-4517-b943-4d19c76b5da9

After:
error

### CODE REVIEW

File: packages/ra-ui-materialui/src/layout/Error.tsx [hunk 1]
```diff
@@ -84,11 +84,14 @@ const Error = (props: ErrorProps): JSX.Element => {
                 <div>{translate('ra.message.error')}</div>
                 {process.env.NODE_ENV !== 'production' && (
                     <>
+                        <Typography className={classes.panel}>
```
Comment (fzaninotto) [inline packages/ra-ui-materialui/src/layout/Error.tsx:87]: you shouldn't use the same class for two distinct elements.

**Review by fzaninotto — CHANGES_REQUESTED** [review]
Not fan of the design.

Here is an alternative suggestion: the AccordionSummary has a fixed title (like "Error Stack") in your PR, and the AccordionDetails contains both the error (in bold) and the error stack. 

It should make it copyable, no?

**Comment by WiXSL** [comment]
> Not fan of the design.
> 
> Here is an alternative suggestion: the AccordionSummary has a fixed title (like "Error Stack") in your PR, and the AccordionDetails contains both the error (in bold) and the error stack.
> 
> It should make it copyable, no?

The thing with this idea is that you are showing an error and there is no actual error to display, until the user interacts with it

**Comment by WiXSL** [comment]
closed-error
opened-error

How do you like it?

**Comment by WiXSL** [comment]
The other option is to add css `user-select : 'all'` to the summary to solve the problem.

chrome-capture

**Review by fzaninotto — APPROVED** [review]
I prefer that solution! 

Note that we've adopted another solution in `next` (see 19d7a69efb0b903e0dcdd5ef43454e46bf253851), but I think your solution is better and should be backported.

**Pull request merged**